config/webpack/environment.js

```diff
@@ -8,27 +8,7 @@
 
 // module.exports = environment;
 
+const { environment } = require('@rails/webpacker');
+const webpack = require('webpack');
 
-const { environment } = require('@rails/webpacker')
-const webpack = require('webpack')
-
-environment.plugins.append(
```

Comment:
These are deprecated in v4 in favour of the optimizations section. By default, webpack 4 generates a vendors bundle now, so no need for a configuration for this.